recipe/meta.yaml

```diff
@@ -75,13 +76,42 @@ outputs:
         - m2-texinfo                         # [win]
         - m2-curl                            # [win]
         - m2-p7zip                           # [win]
-        - texlive-core                       # [unix]
+        #- texlive-core                       # [unix]
+        - texinfo                            # [unix]
         - openjdk
         - {{ posix }}autoconf
         - {{ posix }}automake-wrapper        # [win]
         - {{ posix }}automake                # [not win]
         - make                               # [not win]
         - libcurl                            # [not win]
+        - sed                                # [linux]
+        - gfortran_impl_{{ build_platform }}  # [build_platform != target_platform]
```

Comment:
This is unnecessary

Reply:
No, this is required, in contrast to `gxx` and `gcc`, we don't add the `build_platform` to `gfortran_{{ ctng_target_platform }}`: https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/6290f55b73a887086b0efe0cdb32553646696a8a/recipe/meta.yaml#L93

Reply:
Ah, we do specify it though in `gfortran_osx-64-feedstock`. I'll remove it here and can open an issue/PR on ctng-compiler-activation as we only need `osx-*` support for now.